dockers/docker-ptf/Dockerfile.j2

```diff
@@ -97,10 +97,10 @@ RUN rm -rf /debs \
 
 ## Adjust sshd settings
 RUN mkdir /var/run/sshd \
- && echo 'root:root' | chpasswd \
- && sed -ri 's/^PermitRootLogin\s+.*/PermitRootLogin yes/' /etc/ssh/sshd_config \
- && sed -ri 's/UsePAM yes/#UsePAM yes/g' /etc/ssh/sshd_config \
- && sed -i '$aUseDNS no' /etc/ssh/sshd_config
+    && echo 'root:root' | chpasswd \
+    && sed -ri '/^#?PermitRootLogin/c\PermitRootLogin yes' /etc/ssh/sshd_config \
+    && sed -ri '/^#?UsePAM/c\UsePAM no' /etc/ssh/sshd_config \
+    && sed -ri '/^#?UseDNS/c\UseDNS no' /etc/ssh/sshd_config
```

Comment:
What if space is inserted after the `#` character in the future default sshd_config file, for example:
```
# PermitRootLogin prohibit-password
# UsePam yes
# UseDNS yes
```
Then the sed commands would not be able to update these configurations.

Reply:
@wangxin i think it's not a case here. This is out of box config file and it's common Linux practise to have default configs in such notation.

Reply:
I think this solution is acceptable. However, you could also potentially utilize augtool as we do for modifying sshd_config in the base image here: https://github.com/Azure/sonic-buildimage/blob/master/build_debian.sh#L284